src/applications/pact-act/containers/questions/agent-orange/AgentOrange-2-2-A.jsx

```diff
@@ -82,11 +82,11 @@ const mapDispatchToProps = {
 
 Orange22A.propTypes = {
   formResponses: PropTypes.object.isRequired,
-  setOrange22A: PropTypes.func.isRequired,
-  viewedIntroPage: PropTypes.bool.isRequired,
   router: PropTypes.shape({
     push: PropTypes.func,
-  }),
+  }).isRequired,
```

Comment:
We had some inconsistency in `router` prop type declarations. This change is in the name of consistency.